src/battle-animations.ts

```diff
@@ -794,6 +794,10 @@ class BattleScene {
 			let lombreCount = 0;
 			for (let i = 0; i < side.pokemon.length; i++) {
 				let pokemon = side.pokemon[i];
+				if (pokemon.speciesForme === 'Xerneas-*') {
+					side.pokemon[i].name = 'Xerneas-Neutral';
```

Comment:
I believe nicknames conventionally never include the forme.

```suggestion
					side.pokemon[i].name = 'Xerneas';
```